src/test/config.sol

```diff
@@ -96,17 +96,17 @@ contract Config {
         // Add spells if there is a need to test prior to their cast() functions
         // being called on-chain. They will be executed in order from index 0.
         address[] memory prevSpells = new address[](1);
-        prevSpells[0] = address(0xB70254F2C2C342fB3B168cDa1E43db1E43Ab8DE1);
+        prevSpells[0] = address(0x9E16c8B4C998604471EA0e63ECBb6d6d30F07fA0);
```

Comment:
I'm not sure if this needs to be on, if the tests pass without it I think we should be OK. You'd know better than me whether the tests pass with this set to the default empty value, which is:

```
// prevSpells[0] = <addr>;
```

As context, this was actually added in a previous emergency spell, so there may be context I'm missing here, but to run my local tests with this commented out would hold up this spell so I figured I'd raise it here.

Reply:
Tests don't pass because the spell has not being cast and the chainlog version has been bumped. The way to make it pass is either by setting previous spells or by changing the chainlog version. The latter is more convoluted.

Reply:
This makes sense, thank you. OK to leave it in, resolving this issue.